README.md

```diff
@@ -1,19 +1,35 @@
 # Metabase Firebolt Driver
 
+[Firebolt documentation page](https://docs.firebolt.io/Guides/integrations/metabase.html)
+
+## Overview
+
+
 ## Installation
 
-Please follow Firebolt's [documentation](https://docs.firebolt.io/integrations/business-intelligence/connecting-to-metabase.html) to install the Metabase Firebolt driver on your Metabase instance. If you are using Metabase Cloud, no installation is required since the driver comes preinstalled.
+### Get the driver
+Download the latest driver jar from the [releases page](https://github.com/firebolt-db/metabase-firebolt-driver/releases). Make sure to check the [compatibility matrix](#compatibility-matrix) to get the right version.
+or
```

Comment:
I would highlight **or** here as it might get confusing with users attempting to build the connector.